apps/libcamera_raw.cpp

```diff
@@ -28,8 +28,9 @@ class LibcameraRaw : public LibcameraEncoder
 static void event_loop(LibcameraRaw &app)
 {
 	VideoOptions const *options = app.GetOptions();
-	std::unique_ptr<Output> output = std::unique_ptr<Output>(Output::Create(options));
+	std::shared_ptr<Output> output = std::shared_ptr<Output>(Output::Create(options));
```

Comment:
Just wondering if this needs to be a shared_ptr? We're passing a raw pointer to MetadataReady so I wouldn't have thought anyone is grabbing an extra reference?

Reply:
Yes, you're right, I'll rectify that